0_portfolio_test.R

```diff
@@ -208,7 +208,7 @@ aggregate_map_data <- function(portfolio){
   
   portfolio <- portfolio %>% 
     ungroup() %>% 
-    group_by(!!!rlang::syms(grouping_variables), allocation, 
+    group_by(!!!rlang::syms(grouping_variables), #allocation, 
```

Comment:
So this change doesn't make sense to me - Was this causing a problem? Because I am fairly certain that we need to also group by allocation here. This is just for the mapping data..